Examples/Temperature_Salinity_Diagram.ipynb

```diff
@@ -30,9 +30,11 @@
    "outputs": [],
```

Comment:
<p>This looks like a stray debugging thing that got committed - should be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
remove it!